src/app/src/test/kotlin/org/imperial/mrc/mint/helpers/JSONValidator.kt

```diff
@@ -0,0 +1,93 @@
+package org.imperial.mrc.mint.helpers
```

Comment:
This class copied from HINT - one difference is that hintr has a Response schema for the whole response enclosing `data` which the validator checks for. This isn't in mintr yet.